### ISSUE TITLE
error acessing weights of gmm

### ISSUE THREAD

**Issue by fgvbrt** [pykaldi/pykaldi#30]
hi, I'am trying  to access weights  of trained gmm model. With this code
```
from kaldi.gmm import FullGmm
gmm = FullGmm()
with xopen('final.ubm') as f:
    gmm.read(f.stream(), f.binary)
gmm.weights()
```
but get this error below. I am using latest master kaldi version. Can you help please tell how to acess gmm weights?

Error:
```
>NameError                                 Traceback (most recent call last)
~/anaconda/envs/kaldi/lib/python3.5/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

>~/anaconda/envs/kaldi/lib/python3.5/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    393                             if callable(meth):
    394                                 return meth(obj, self, cycle)
--> 395             return _default_pprint(obj, self, cycle)
    396         finally:
    397             self.end_group()

>~/anaconda/envs/kaldi/lib/python3.5/site-packages/IPython/lib/pretty.py in _default_pprint(obj, p, cycle)
    508     if _safe_getattr(klass, '__repr__', None) is not object.__repr__:
    509         # A user-provided repr. Find newlines and replace them with p.break_()
--> 510         _repr_pprint(obj, p, cycle)
    511         return
    512     p.begin_group(1, '<')

>~/anaconda/envs/kaldi/lib/python3.5/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    699     """A pprint that just redirects to the normal repr function."""
    700     # Find newlines and replace them with p.break_()
--> 701     output = repr(obj)
    702     for idx,output_line in enumerate(output.splitlines()):
    703         if idx:

>~/anaconda/envs/kaldi/lib/python3.5/site-packages/pykaldi-0.0.9-py3.5-linux-x86_64.egg/kaldi/matrix/__init__.py in __repr__(self)
    553 
    554     def __repr__(self):
--> 555         return str(self)
    556 
    557     def __str__(self):

>~/anaconda/envs/kaldi/lib/python3.5/site-packages/pykaldi-0.0.9-py3.5-linux-x86_64.egg/kaldi/matrix/__init__.py in __str__(self)
    562         # https://github.com/pytorch/pytorch/blob/master/torch/tensor.py
    563         if sys.version_info > (3,):
--> 564             return _str._vector_str(self)
    565         else:
    566             if hasattr(sys.stdout, 'encoding'):

>NameError: name '_str' is not defined"
```

**Comment by vrmpx** [comment]
Hi @fgvbrt, 

I see that you are using Anaconda and Python 3.5.
In order to replicate this problem, would you mind telling me:

1) what OS are you using? 
2) what anaconda version are you using?

Thanks

**Comment by fgvbrt** [comment]
Hi @vrmpx, please find details below:
1) Ubuntu 16.04.4
2) conda 4.3.29, anaconda Command line client (version 1.5.1), for installation I used  conda environments with Python 3.5.5

**Comment by dogancan** [comment]
Hmm, scratch that renaming theory. It was github's formatting which turned those double underscores in the error message to bold text.

**Comment by dogancan** [comment]
@fgvbrt I have been looking into why this happened but I couldn't find anything online and we cannot replicate it on our end. The reason I haven't merged #31 yet is because I want to understand the reason why this is an issue on your end only.

Could you help us debug this by running these [test scripts](https://gist.github.com/dogancan/f03b5d153714a7c7285a3cea06dca296/archive/a2f31a8b1f9fba98f20f82be85fb28cf2be9cdac.zip) on your system and sending us the output? Please run them as scripts, i.e. do not type them into an ipython session, so we can make sure no other packages are interfering with their operation. Also make sure you run them against vanilla PyKaldi, i.e. without the change in #31.

Here is the output I am getting
```
>>> python test1.py

 1  2
 3  4
[kaldi.matrix.Matrix of size 2x2]

```
```
>>> python test2.py

 0
 0
 0
 0
[kaldi.matrix.SubVector of size 4]

```

**Comment by fgvbrt** [comment]
Hi @dogancan  I am getting the same  output. And with latest master I also can't reproduce the error in first message. Thanks for help.
